<launch>
    <arg name="id" default="254" />
    <arg name="swarm_num" default="$(env swarm_num)" doc="swarm_num type [1, 2, 3]"/>
    <arg name="broadcast" default="$(env broadcast)" doc="broadcast type [true, false]"/>

    <node pkg="radio_proxy" type="GS_proxy_node" name="GS_proxy" output="screen">
        <param name="id" value="$(arg id)"/>
        <!-- <param name="port" value="/dev/usb_radio" /> -->
        <param name="port" value="/dev/ttyUSB0" />
        <param name="baud_rate" value="230400" />
        <param name="swarm_num" value="$(arg swarm_num)" />
        <param name="broadcast" value="$(arg broadcast)" />
    </node>
</launch>